aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -73,9 +76,11 @@ index_select_add(const Tensor &select_indices,
   auto output_stride1 = output.stride(1);
 
   for (int64_t i = 0; i < numel; i++) {
-    THBlas_axpy<data_t>(ddim, 1,
-            src_data + src_stride0 * select_indices_data[i], src_stride1,
-            output_data + output_stride0 * add_indices_data[i], output_stride1);
+    if (select_indices_data[i] != padding_idx) {
```

Comment:
For those of us who are not familiar with embedding bag or what the padding_idx argument does (it's documented very cryptically), what's going on here?

A brief comment about why padding_idx is skipped would be helpful here, too

Reply:
Does the comment I added make sense?

Perhaps we can make the documentation less cryptic